src/transformers/models/vilt/modeling_vilt.py

```diff
@@ -1215,12 +1219,10 @@ def forward(
         >>> processor = ViltProcessor.from_pretrained("dandelin/vilt-b32-finetuned-coco")
         >>> model = ViltForImageAndTextRetrieval.from_pretrained("dandelin/vilt-b32-finetuned-coco")
 
-        >>> # prepare inputs
-        >>> encoding = processor(image, text, return_tensors="pt")
-
```

Comment:
`text` is not defined here. And it seems this is not used, because the inputs is prepared in the loop below.

Reply:
Thanks for fixing